README.md

```diff
@@ -52,6 +52,8 @@ You must set certain values in the `config.ini` file, in particular your Discor
     * `DownvoteEmoji`: The name of your downvote emoji.
     * `ModeratorRole`: The name of the role that your moderators have (for functions like DMing users).
     * `DeveloperRole`: The name of the role that your developers have (for functions like restarting the bot). This could be the same role than moderator.
+    * `PenaltyRole`: The name of the role used for placing users in a "penalty box" (which presumably removes some 
```

Comment:
i think this should be optional. right now, if you dont have the penaltyrole, you get the exception mis-configured penalty role everytime someone leaves

Reply:
06e0fe4